.travis.yml

```diff
@@ -35,6 +32,10 @@ matrix:
     - python: 3.5
       env: TOXENV=flake8
     - python: 3.5
+      env: TOXENV=coverage
+      addons:
+        firefox: latest
+    - python: 3.5
```

Comment:
Can you rebase again? This pull should only contain changes on top of current master.